### DISCUSSION TITLE
Build failed time limit

### DISCUSSION THREAD

**Discussion by dilsher07singh** [vercel/community#3991] [Feature Question]
### Question

Your deployment's build step did not complete within the maximum of 45 minutes (task 6b4e39b1a35b482dbf05a26796e42c5a). I need help with this. Is there no way to increase the time as we need to deploy this quick

**Comment by amyegan** [comment]
Hey @dilsher07singh. There is a [build Time per deployment](https://vercel.com/docs/limits/overview#build-time-per-deployment) limit of 45 minutes for all plan types. This cannot be increased. 

Please consider other solutions, such as [Incremental Static Regeneration (ISR)](https://vercel.com/docs/incremental-static-regeneration#), which can decrease your project's build time by generating some pages on-demand.
